types/field.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+// FieldDefinition is a representation of a GraphQL FieldDefinition.
+//
+// http://spec.graphql.org/draft/#FieldDefinition
+type FieldDefinition struct {
+	Alias      Ident
```

Comment:
FieldDefinitions should not have an Alias.

Reply:
good catch. fixed in 6537f29